cwms-data-api/src/main/java/cwms/cda/data/dao/LocationLevelsDaoImpl.java

```diff
@@ -587,7 +587,7 @@ public TimeSeries retrieveLocationLevelAsTimeSeries(ILocationLevelRef levelRef,
         CWMS_ENV_PACKAGE.call_SET_SESSION_OFFICE_ID(dsl.configuration(), officeId);
         ZTSV_ARRAY locLvlValues = CWMS_LEVEL_PACKAGE.call_RETRIEVE_LOC_LVL_VALUES3(dsl.configuration(),
                 specifiedTimes, locationLevelId, levelUnits, attributeId, attributeValue, attributeUnits,
-                "UTC", officeId);
+                "UTC", officeId, "VN");
```

Comment:
I don't entirely remember making that change but it looks like it was related to updating the jooq codegen.  The method signature now has a p_level_precedence parameter and the procedure def lists "VN" as the default. 

Here's the doc on it:  
 @param p_level_precedence   One or two characters that specify the precedence of virtual and non-virtual location levels in the results.
 *                             The default value 'VN' gives virtual location levels higher precedence, allowing them to hide non-virtual location levels
 *                             where the two types overlap in time. Valid value are:
 * <ul>
 *   <li><b>N</b> specifies results from non-virtual (normal) location levels only
 *   <li><b>V</b> specifies results from virtual location levels only
 *   <li><b>NV</b> specifies results from non-virtual (normal) location levels where they exist, with virtual location levels allowed where non-virtual levels don't exist
 *   <li><b>VN</b> (default) specifies results from virtual location levels where they exist, with non-virtual location levels allowed where virtual levels don't exist
 * </ul>

Reply:
I could change it to null - thats probably the more correct thing to do.

Reply:
no, passing null doesn't seem to trigger the default.  not sure how to make the call from jooq without specifying the parameter so that the default will take effect.

Reply:
I'd leave it in just put a short comment about the behavior and where to find more information. I doubt I'm the only one that might get confused by that.